video/external/src/decoder/openh264.rs

```diff
@@ -1,13 +1,62 @@
 use core::slice;
 use std::ffi::{c_int, c_uchar};
+use std::fmt::Display;
 use std::ptr;
+use std::sync::Arc;
 
 use crate::decoder::openh264_sys::{self, videoFormatI420, ISVCDecoder, OpenH264};
 use crate::decoder::VideoDecoder;
 
 use ruffle_render::bitmap::BitmapFormat;
 use ruffle_video::error::Error;
 use ruffle_video::frame::{DecodedFrame, EncodedFrame, FrameDependency};
+use thiserror::Error;
+
+#[derive(Debug, PartialEq, Eq)]
+pub struct H264Version(u32, u32, u32);
+
+impl Display for H264Version {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "({},{},{})", self.0, self.1, self.2)?;
```

Comment:
Why make it look like a tuple, instead of a "normal" version number? :thinking:
I mean, `(2,4,1)` -> `2.4.1`?

Reply:
:facepalm: of course, I just replaced the tuple with the struct without really thinking about it